stable/redis-ha/templates/redis-ha-statefulset.yaml

```diff
@@ -47,8 +48,8 @@ spec:
         - sh
         args:
         - /readonly-config/init.sh
-        env:
 {{- if .Values.auth }}
+        env:
```

Comment:
this will make `env:` removed when `.Values.auth` is `false`.
```yaml
        args:
        - "/readonly-config/init.sh"
        - name: POD_IP
          valueFrom:
            fieldRef:
              fieldPath: status.podIP
```
This line should go back to L50.

Reply:
Ahh good find! I believe me intention was to delete the POD_IP var as it is no longer necessary. Thank you I will fix this